### PR TITLE
Replace unicode characters with ASCII

### PR DESCRIPTION
**Pull request by mgiuffrida**
Unicode characters in source code screw up some build systems, like Chromium.

### CODE REVIEW

**Comment by mgiuffrida** [comment]
googlebot: oops, michaelpg@chromium.org is my chromium identity. this is also me.

File: test/lib/brick.js [hunk 1]
```diff
@@ -1833,8 +1833,8 @@ if (typeof WeakMap === "undefined") {
     var LEFT_MOUSE_BTN = 0;
     var GET_DEFAULT_LABELS = function() {
         return {
-            prev: "←",
```
Comment (vicb) [inline test/lib/brick.js]: Coulf you please revert this change, `brick.js` is a 3rd party lib.
As it is used in tests only, it should not affect you ?
